packages/investor-idle/src/IdleSdk.ts

```diff
@@ -1,14 +1,12 @@
 import axios from 'axios'
 
-import { IdleVault } from './constants'
+import { apiKey, baseUrl, IdleVault } from './constants'
 
 export class IdleSdk {
   async getVaults(): Promise<IdleVault[]> {
-    const results = await axios.get(
-      'https://api.idle.finance/pools?api-key=bPrtC2bfnAvapyXLgdvzVzW8u8igKv6E'
-    )
+    const results = await axios.get(`${baseUrl}/pools?api-key=${apiKey}`)
```

Comment:
Either remove trailing slash here or in constants file 
```suggestion
    const results = await axios.get(`${baseUrl}pools?api-key=${apiKey}`)
```